src/components/LabeledSlider.tsx

```diff
@@ -1,17 +1,16 @@
-import { InputNumber, Slider } from "antd";
+import { InputNumber, Slider, SliderSingleProps } from "antd";
+import { SliderRangeProps } from "antd/es/slider";
```

Comment:
why was this import location needed? It's not exposed in the main package?

Reply:
Unfortunately not... I will move the `SliderSingleProps` down with it though for consistency.